deploy/cloudbuild.yaml

```diff
@@ -3,6 +3,9 @@ steps:
   - name: "gcr.io/cloud-builders/docker"
     args: ["build", "-f", "deploy/Dockerfile",
            "-t", "gcr.io/kritis-project/kritis-server:${COMMIT_SHA}", "."]
+  - name: "gcr.io/cloud-builders/docker"
+    args: ["build", "-f", "deploy/Dockerfile",
+           "-t", "gcr.io/kritis-project/kritis-server:latest", "."]
```

Comment:
You might need to add `gcr.io/kritis-project/kritis-server:latest` to `images` at the bottom so it's pushed at the end

Reply:
done